maps/atlas.dmm

```diff
@@ -320,6 +320,11 @@
 /obj/machinery/door/poddoor/blast/pyro{
 	id = "qm_dock_out"
 	},
+/obj/forcefield/energyshield/perma/doorlink{
+	desc = "A door-linked force field that prevents gasses from passing through it.";
+	name = "Door-linked Atmospheric Forcefield";
+	powerlevel = 1
+	},
```

Comment:
Setting `powerlevel` to `1` on these doorlink shields isn't actually doing anything to affect their ability to block liquids. You instead would need to make a new child of `/obj/forcefield/energyshield/perma/doorlink` that doesn't have the `ALWAYS_SOLID_FLUID` flag to make these atmos only forcefields, and if we want to be consistent with powerlevels then you would need to override `setactive` to set the `powerlevel` to `1`

Side note, if these are just going to be for atmos then I feel like they should be colored `#3333FF` instead to  be consistent with the other atmos forcefields from the generator

Reply:
Uh, thing is, every map that already has those forcefields (donut 3, kondaru, horizon, cog1) has the exact same ones. All I did was using the existing atmos forcefields and just adding the missing ones, if you'll take a look for, say, donut 3, you'll see the same code lines

So the suggested changes would also need to include the maps that already have the implemented forcefields :/ And this wasn't meant to be the goal of this PR